src/expr/src/scalar/func/impls/date.rs

```diff
@@ -103,6 +106,10 @@ impl<'a> EagerUnaryFunc<'a> for ExtractDate {
     fn output_type(&self, input: ColumnType) -> ColumnType {
         ScalarType::Numeric { max_scale: None }.nullable(input.nullable)
     }
+
+    fn is_monotone(&self) -> bool {
+        most_significant_unit(self.0)
```

Comment:
Most of these is_monotone functions seem uncovered, this one seems to do something interesting.

Reply:
I've added the `date` type to the equivalence test in `interpret.rs`, which should give us a little coverage.

And yeah, not much test coverage for the annotations. The nicest test to write for this would be some property test that, for every function, generated some random inputs and verified that the function was behaving monotonically if it claimed to be monotonic... but right now this sort of test is very expensive to write. (And the easy tests to write -- which just assert that a particular `is_monotone()` call returns true or whatever -- don't seem especially useful.)